_pages/docs/next-ls/editors.md

```diff
@@ -14,11 +14,30 @@ github_stars: elixir-tools/next-ls
 
 ## Neovim
 
+### [elixir-tools.nvim](https://github.com/elixir-tools/elixir-tools.nvim)
+
 [elixir-tools.nvim](https://github.com/elixir-tools/elixir-tools.nvim) is a first party elixir-tools editor plugin and will install and manage Next LS for you
 
+### [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#nextls)
```

Comment:
Remove the links from the headers. There are already links in the body of each section.